lms/djangoapps/teams/static/teams/templates/edit-team.underscore

```diff
@@ -0,0 +1,33 @@
+<div class="create-team wrapper-msg is-incontext urgency-low warning is-hidden">
+    <div class="msg">
+        <div class="msg-content">
+            <h3 class="title"><%- gettext("Oops!") %></h3>
```

Comment:
Hi @muhammad-ammar. "Your team could not be created" should be the ``h3`` only if the message applies to "create team" errors. If it applies to ALL error messages then we should come up with something else.